lib/extensions/ar_virtual.rb

```diff
@@ -283,22 +284,22 @@ def without_virtual_includes!
       self
     end
 
-    def find_with_associations_with_virtual
-      recs = without_virtual_includes.send(:find_with_associations_without_virtual)
+    include Module.new {
+      # From ActiveRecord::FinderMethods
+      def find_with_associations
+        recs = without_virtual_includes.method(:find_with_associations).super_method.call
```

Comment:
Ditto. Alternatives? Only way I could think of doing it with the 'without_virtual includes' way of passing self.